Modules/Core/Common/include/itkMultiThreaderBase.h

```diff
@@ -466,3 +466,50 @@ ITKCommon_EXPORT std::ostream &
 
 } // end namespace itk
 #endif
+
+#ifndef ITK_TEMPLATE_EXPLICIT_MultiThreaderBase
+
+namespace itk
+{
+extern template void ITKCommon_EXPORT_EXPLICIT
+                     MultiThreaderBase::ParallelizeImageRegion<1>(const ImageRegion<1> &, TemplatedThreadingFunctorType<1>, ProcessObject *);
```

Comment:
Does code completion pick up the previous definition with named parameters? If not, we should add parameter names here.